assets/scripts/app/__tests__/NoConnectionMessage.test.js

```diff
@@ -1,44 +1,51 @@
 /* eslint-env jest */
 import React from 'react'
-import { shallow } from 'enzyme'
-import { mountWithIntl as mount } from '../../../../test/helpers/intl-enzyme-test-helper.js'
-import { NoConnectionMessage } from '../NoConnectionMessage'
+import { fireEvent } from '@testing-library/react'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
+import NoConnectionMessage from '../NoConnectionMessage'
```

Comment:
One thing that we did previously to avoid having a Redux wrapper is to import the unwrapped, unconnected React component to test. We no longer do that in react-testing-library because our Redux helper works really well. So when we port the test suite, we also change the import to the default export of ` '../NoConnectionMessage'`, which is the Redux-connected component.